plugins/importexport/native/filter/NativeFilterHelper.inc.php

```diff
@@ -224,6 +258,66 @@ function parsePublicationCover($filter, $node, $object) {
 
 						file_put_contents($filePath, base64_decode($n->textContent));
 						break;
+					case 'href':
+						if (!isset($coverImage['uploadName'])) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.coverImageNameUnspecified'));
+							break;
+						}
+
+						import('classes.file.PublicFileManager');
+						$publicFileManager = new PublicFileManager();
+						$filePath = $publicFileManager->getContextFilesPath($context->getId()) . '/' . $coverImage['uploadName'];
+						$allowedFileTypes = ['gif', 'jpg', 'png', 'webp'];
+						$extension = pathinfo(strtolower($filePath), PATHINFO_EXTENSION);
+						if (!in_array($extension, $allowedFileTypes)) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.invalidFileExtension'));
+							break;
+						}
+
+						$imageUrl = $n->getAttribute('src');
+						if (empty($imageUrl)) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.missingHref'));
+							break;
+						}
+
+						$fileContents = file_get_contents($imageUrl);
+						if ($fileContents === false) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.failedDownloadingFile', array('url' => $imageUrl)));
+							break;
+						}
+
+						file_put_contents($filePath, $fileContents);
+						break;
+					case 'file_url':
+						if (!isset($coverImage['uploadName'])) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.coverImageNameUnspecified'));
+							break;
+						}
+
+						import('classes.file.PublicFileManager');
+						$publicFileManager = new PublicFileManager();
+						$filePath = $publicFileManager->getContextFilesPath($context->getId()) . '/' . $coverImage['uploadName'];
+						$allowedFileTypes = ['gif', 'jpg', 'png', 'webp'];
+						$extension = pathinfo(strtolower($filePath), PATHINFO_EXTENSION);
+						if (!in_array($extension, $allowedFileTypes)) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.invalidFileExtension'));
+							break;
+						}
+
+						$imageUrl = $n->textContent;
+						if (empty($imageUrl)) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.missingFileUrl'));
+							break;
+						}
+
+						$fileContents = file_get_contents($imageUrl);
+						if ($fileContents === false) {
+							$deployment->addWarning(ASSOC_TYPE_PUBLICATION, $object->getId(), __('plugins.importexport.common.error.failedDownloadingFile', array('url' => $imageUrl)));
+							break;
+						}
+
+						file_put_contents($filePath, $fileContents);
+						break;
```

Comment:
This whole piece can be removed, once you copy the implementation for the `<href>`

```suggestion
```